src/index.ts

```diff
@@ -1 +1,52 @@
-export const checkDangers = ()=>{}
\ No newline at end of file
+/* eslint-disable @typescript-eslint/no-shadow */
+// Provides dev-time typing structure for  `danger` - doesn't affect runtime.
+// https://github.com/danger/danger-js/blob/main/docs/usage/extending-danger.html.md#writing-your-plugin
+import * as TE from "fp-ts/TaskEither";
+import * as E from "fp-ts/Either";
+
+import { pipe } from "fp-ts/function";
+
+import { DangerDSLType } from "../node_modules/danger/distribution/dsl/DangerDSL";
```

Comment:
can't we import this without a full reference to node_modules?

Reply:
From [here](https://github.com/danger/danger-js/blob/main/docs/usage/extending-danger.html.md#writing-your-plugin): Note, that you should not add a import { danger, et, cetera } from "danger" in your plugin. The dirty truth is that Danger completely ignores that at runtime, and just puts all of it's exports inside the global context. You'll get compiler errors, or confusing shadow variable issues. So assume that it's there, and working fine.